bin/dart_doc_syncer.dart

```diff
@@ -3,22 +3,40 @@ import 'dart:async';
 import 'package:logging/logging.dart';
 
 import 'package:dart_doc_syncer/documentation_updater.dart';
+import 'package:dart_doc_syncer/example2uri.dart';
+import 'package:dart_doc_syncer/options.dart';
 
 /// Syncs an example application living in the angular.io repository to a
 /// dedicated repository that will contain a generated cleaned-up version.
 ///
-///     dart_doc_syncer <examplePath> <exampleRepository>
-Future main(List<String> args) async {
-  Logger.root.level = Level.WARNING;
+///  dart_doc_syncer [-h|-n|-v] [<exampleName> | <examplePath> <exampleRepo>]
+Future main(List<String> _args) async {
```

Comment:
It's just an idiom I have been using. Generally speaking it leaves the original `_args` available (untouched) in the rest of the body. (And yes, I am aware that for global and member declarations the underscore means private.) 

Let me know if in this case if you prefer that I just use `args` and I'll change it.